### PR TITLE
[Relay][Frontend][ONNX] Add ConvInteger support.

### PR DESCRIPTION
**Pull request by jwfromm**
While playing with [onnxruntime's quantization tools](https://www.onnxruntime.ai/docs/how-to/quantization.html) I found that `ConvInteger` nodes were heavily used and pretty easy to map to `qnn.conv2d`. This PR adds support for that op.

### CODE REVIEW

**Comment by jwfromm** [comment]
@mbrookhart @AndrewZhaoLuo can you guys take a look at this PR?

**Comment by mbrookhart** [comment]
I'm having trouble getting the docker images in CI updated due to a TF update.

**Comment by jwfromm** [comment]
@mbrookhart that makes sense, I've removed all changes to convtranspose from this PR.

**Review by mbrookhart — APPROVED** [review]

File: python/tvm/relay/frontend/onnx.py [hunk 1]
```diff
@@ -3193,6 +3196,79 @@ def get_scalar(x, dtype="float32"):
...
+                pass
+            else:
+                msg = 'Value {} in attribute "auto_pad" of operator Conv is invalid.'
+                raise tvm.error.OpAttributeInvalid(msg.format(attr["auto_pad"]))
+            attr.pop("auto_pad")
+
+        out_channels = kernel_shape[0]
+        dilation = attr.get("dilations", [1] * (ndim - 2))
+        strides = attr.get("strides", [1] * (ndim - 2))
+        padding = attr["pads"] if "pads" in attr else 0
```
Comment (AndrewZhaoLuo) [inline python/tvm/relay/frontend/onnx.py:3248]: Should we throw if both pads and auto_pad are present?

Comment (jwfromm) [reply python/tvm/relay/frontend/onnx.py:3248]: Good question, this is the same logic used in all other convolutions though (where we overwrite pads with auto_padding if both are present). I personally think that defaulting to auto_pad if present is reasonable behavior but am open to other takes. @mbrookhart what do you think?

Comment (AndrewZhaoLuo) [reply python/tvm/relay/frontend/onnx.py:3257]: Thought about it a bit more. So the onnx spec has it so you cannot have both. I think it's safe to assume the converter will only hit well formed onnx models so I think this suggested default behavior is fine.

**Review by AndrewZhaoLuo — APPROVED** [review]

**Comment by mbrookhart** [comment]
Thanks @jwfromm @AndrewZhaoLuo

**Pull request merged**